File: beginner/pagina10/2770/2770.cpp
#include <iostream>

using namespace std;

int main(){

    int lp, ap, qnt;
    
    while(cin >> lp >> ap >> qnt){
        for(int i = 0; i < qnt; i++){
            int lpc, apc;
            cin >> lpc >> apc;

            bool condicoes = (
                lpc <= lp && apc <= ap || lpc <= ap && apc <= lp
            )

            if(condicoes){
                cout << "Sim" << endl;
            } else {
                cout << "Nao" << endl;
            }
        }
    }

    
    return 0;
}

File: beginner/pagina10/2770/2770.php
<?php 

    $output = "";

    while($input = readline()){
        list($lp, $ap, $qnt) = explode(" ", $input);

        $lp = (int) $lp; $ap = (int) $ap; $qnt = (int) $qnt;

        for($i = 0; $i < $qnt; $i++){
            list($lpc, $apc) = explode(' ', readline());
            
            $lpc = (int) $lpc; $apc = (int) $apc;

            if(($lpc <= $lp && $apc <= $ap) || ($lpc <= $ap && $apc <= $lp)){
                $output .= "Sim\n";
            } else {
                $output .= "Nao\n";
            }

        }
    }

    echo "$output";
?>